blazor/datagrid/batch-editing.md

```diff
@@ -180,6 +180,94 @@ In the following example:
 
 ```
 
+## Event trace while Batch editing
```

Comment:
this is the example how the events topics should be made

https://ej2.syncfusion.com/angular/documentation/grid/grouping/grouping#grouping-events